schema/m3-region.schema.json

```diff
@@ -221,6 +221,12 @@
                             "title": "Is it usable coming in?",
                             "default": true
                           },
+                          "openEnd": {
+                            "$id": "#/items/properties/nodes/items/properties/sparking/properties/runways/items/properties/openEnd",
+                            "type": "integer",
+                            "title": "Does the runway have an open end beside it (as opposed to a wall)?",
+                            "default": 1
```

Comment:
I don't think this needs a default value. Same for the other one.

Reply:
This default saves several hundred lines of code.

Reply:
I'd argue having default values hides several hundred lines of real data, that I assume developers then have to fetch in some other schema file that they probably don't care about and probably don't want to bother with.